### PR TITLE
docs: update Ehis blog

### PR DESCRIPTION
**Pull request by Michaelg22**
Signed-off-by: Michaelg22 <mike.giroux22@outlook.com>

### Related Issue

Solving Issue: #xxx

### PR Type

- [ ] Fix typos or format(punctuation, space, indentation, code block, etc.)
- [ ] Update inappropriate or misunderstanding content
- [ ] Add missing content or new documents
- [ ] Delete useless content or outdated documents
- [ ] Localize or translate documents
- [ ] Deal with GitHub action files and scripts
- [ ] Deal with chores

### CODE REVIEW

File: blog/2023-01-30-awsbilling-slack-blog/index.md [hunk 1]
```diff
@@ -192,53 +190,41 @@ We want to export our slack sink as an environmental variable so we can easily u
...
+Create a config file for Amazon Billing Source, don't forget the update the access key and secret key obtained from your AWS console (IAM - Programmatic access)
 
-```shell
-http://<ip_address>:<port>/gateway/<eventbus>
+```shell 
+cat << EOF > config.yml
+target: http://192.168.49.2:30001/gateway/billing
+secret:
+  access_key_id: AKIAIOSFODNN7EXAMPLE # <- Your access key
+  secret_access_key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY <- Your secret key
```
Comment (ehisakhile) [inline blog/2023-01-30-awsbilling-slack-blog/index.md:207]: secret_access_key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY #<- Your secret key

**Review by ehisakhile — CHANGES_REQUESTED** [review]
Need to make changes

File: blog/2023-01-30-awsbilling-slack-blog/index.md (hunk 1 above)
Comment (Michaelg22) [reply blog/2023-01-30-awsbilling-slack-blog/index.md:207]: Please make a suggestion in the future not a comment

File: blog/2023-01-30-awsbilling-slack-blog/index.md [hunk 2]
```diff
@@ -186,59 +184,46 @@ kubectl get pods -n vanus
...
 
-Use vim editor to update our config.yml file. Add to the file this line of code:
-```shell
-"target": "http://192.168.49.2:30001/gateway/billing"
+```shell 
+cat << EOF > config.yml
+target: http://192.168.49.2:30001/gateway/billing
+secret:
+  access_key_id: AKIAIOSFODNN7EXAMPLE # <- Your access key
+  secret_access_key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY <- Your secret key
```
Comment (Michaelg22) [inline blog/2023-01-30-awsbilling-slack-blog/index.md:207]:
```suggestion
  secret_access_key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY # <- Your secret key
```

**Review by ehisakhile — APPROVED** [review]

**Pull request merged**